task.sql

```diff
@@ -3,0 +1,14 @@
+USE ShopDB;
+
+-- Create an index for the Customers table to optimize login by Email
+CREATE INDEX Email ON Customers (Email);
+
+-- Create an index for the Products table to optimize product searches by Name
+CREATE INDEX Name ON Products (Name);
```

Comment:
Consider using a more descriptive name for the index on the `Name` column, such as `idx_Products_Name`, to make it clear which table and column the index is associated with.